### PR TITLE
4 - Added tests

### PR DESCRIPTION
**Pull request by timmi05**

### CODE REVIEW

File: __tests__/AppContainer.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+describe('AppContainer rendering', () => {
+    it('renders correctly', () => {
+        const component = shallow(<AppContainer/>);
+        expect(component).toMatchSnapshot();
+    });
+
+    it('renders correctly', () => {
+        const component = shallow(<AppContainer
+        productSelected = { true }
```
Comment (ant-plotnikov) [inline __tests__/AppContainer.spec.jsx]: Could you skip passing truthy values? Just `<AppContainer productSelected />`

File: __tests__/result/panel/ResultSort.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        const component = shallow(<ResultSort
+        productSelected = { false }
+        />);
+        expect(component).toMatchSnapshot();
+    });
+
+    it('simulate click on button by RELEASE DATE', () => {
+        const component = mount(<ResultSort/>);
+        component.find({label: 'RELEASE DATE'}).simulate('click');
+        expect(component.find({className:'button result-sort-button result-sort-left-button'})
```
Comment (ant-plotnikov) [inline __tests__/result/panel/ResultSort.spec.jsx]: Could we have a single expectation for each test case? You are mixing snapshot and functional testing here

File: cypress/integration/app-init.spec.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  it('Loads main page', () => {
+    cy.visit('http://localhost:8080/');
+
+    const inputText = 'Movie name';
+
+    cy.get('.search-panel-input')
+      .type(inputText)
+      .should('have.value', inputText);
+
+    cy.get('.search-panel-button')
```
Comment (ant-plotnikov) [inline cypress/integration/app-init.spec.js]: This one should be a separate test case. To write `visit` code once please move it to `beforeEach` hook

File: __tests__/AppContainer.spec.jsx (hunk 1 above)
Comment (timmi05) [reply __tests__/AppContainer.spec.jsx]: fixed

File: __tests__/result/panel/ResultSort.spec.jsx (hunk 2 above)
Comment (timmi05) [reply __tests__/result/panel/ResultSort.spec.jsx]: fixed

File: cypress/integration/app-init.spec.js (hunk 3 above)
Comment (timmi05) [reply cypress/integration/app-init.spec.js]: fixed

**Pull request merged**